src/static/css/page.css

```diff
@@ -423,10 +427,20 @@ tbody tr:nth-child(even) {
 
 figure .big-number {
   text-align: center;
+  font: bold 128px/1em Poppins, sans-serif;
   font: bold 8rem/1em Poppins, sans-serif;
   color: #62718b;
 }
 
+figure .really-big-number {
+  text-align: center;
+  font: bold 24px/1em Poppins, sans-serif;
+  font: bold 1.5rem/1em Poppins, sans-serif;
```

Comment:
Are we duplicating `font` rule it to ensure proper rendition in browsers that do not support `rem` unit? If so, it looks like the support for `rem` is [pretty good](https://caniuse.com/#feat=rem) now.

Reply:
Yes you are correct that it's rare to find a browser that doesn't support this. However I find it easier to think in `px` values so having it on the line above is helpful to me, with the added bonus of support if needed.

Reply:
Which means you overwrite the rules immediately defined. I think, those performance related tools like lighthouse will not be happy about it.